contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
```

Comment:
Multiple repos: https://github.com/kubernetes/kubernetes/issues/24343

It will take quite a while to be completed.

I'm fine with a statement at the beginning stating that this only applies to the kubernetes repo and other repos are TBD.